apps/mds_docs/src/app/globals.css

```
@@ -0,0 +1,92 @@
+  --spacing-title-to-body: 4px;
+  --spacing-section-gap: 40px;
+  /* Radius tokens */
+  --radius-small: 8px;
+  --radius-button: 12px;
+  --radius-card: 16px;
+  --radius-dialog: 28px;
+  --radius-input: 12px;
+  --radius-badge: 4px;
+  --radius-chip: 100px;
+  /* Typography tokens */
+  --typography-font-family-brand: Pretendard;
```

Stylelint (17.9.0)

[error] Expected "Pretendard" to be "pretendard" (value-keyword-case)

(value-keyword-case)